### ISSUE TITLE
-Wunused-variable in drivers/net/netdevsim/bpf.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#266]
```
drivers/net/netdevsim/bpf.c:557:30: error: unused variable 'state' [-Werror,-Wunused-variable]
        struct nsim_bpf_bound_prog *state;
                                    ^
1 error generated.
```

Patch sent: https://lore.kernel.org/lkml/20181112221042.11160-1-natechancellor@gmail.com/

This is only in -next, I will close this issue once the patch has been accepted and is merged into a -next version.

**Comment by nathanchance** [comment]
Patch accepted: https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git/commit/?id=ac8acec9912a93be9953446766e0bb73aeeecc64

**Comment by nathanchance** [comment]
Fixed as of next-20181119: https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=ac8acec9912a93be9953446766e0bb73aeeecc64
